docs/releases/unreleased.md

```diff
@@ -54,3 +54,8 @@
 ## utils
 
 - Moved `model_selection.expand_param_grid` to `utils.expand_param_grid`.
+
+## compat
+
+- Added `compat.pytorch.PyTorch2RiverClassifier`
+- Refactored `compat.pytorch.PyTorch2RiverRegressor`
```

Comment:
Actually you don't need the `.pytorch.`. It's important to be accurate here because we actually add a link when we build the documentation.